### PR TITLE
dev-calculations Pull Request

### PR DESCRIPTION
**Pull request by FSolM**
Base calculation logic implementation ⚙️

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      return Big(num1).times(num2);
+    case '/':
+      return Big(num1).div(num2);
+    case '%':
+      return Big(num1).mod(num2);
+    default:
+      console.error('Something went wrong');
+      break;
+  }
+};
```
Comment (shloch) [inline src/logic/calculate.js]: You missed out one instruction
- https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/buttonAsArgument.png

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export default (calculator, btnName) => {
+  if (btnName === '+/.') {
+    calculator.total *= -1;
+    calculator.next *= -1;
+  }
+};
```
Comment (shloch) [inline src/logic/operate.js]: You must have reversed the contents of what was supposed to be in `operate,js` and `calculate.js`
Screen Shot 2019-12-09 at 12 50 28 PM

Comment (FSolM) [reply src/logic/operate.js]: oh damn... 😮

File: src/logic/operate.js [hunk 3]
```diff
@@ -1,6 +1,19 @@
...
+export default (num1, num2, operation) => {
+  switch (operation) {
+    case '+':
+      return Big(num1).plus(num2);
+    case '-':
+      return Big(num1).minus(num2);
+    case 'x':
+      return Big(num1).times(num2);
+    case '/':
+      return Big(num1).div(num2);
```
Comment (shloch) [inline src/logic/operate.js]: When using this component on next milestone, dont forget to handle division by ZERO 😊

**Pull request merged**